casts/highlights/src/useOnWheelHandler.ts

```diff
@@ -39,18 +64,28 @@ function wrapVerticalToHorizontalWheelMovementHandler(
         // don't propagate to an outter scroll view if any
         event.stopPropagation();
 
-        const deltaCoord = event.deltaY || event.deltaX;
-        // Note: e.deltaY is not present for `DOMMouseScroll` event (used by Firefox)
```

Comment:
I wonder if this is still valid for Firefox :)

Reply:
I checked on my Firefox browser. There is a regular [wheel](https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event) event.

Reply:
Is the speed of it actually OK? As I see the former hack was applied by multiplying the delta to some factor…

Reply:
I checked on Chrome, Firefox and Safari. The speed is ok.